tools/build-linux-release.sh

```diff
@@ -0,0 +1,27 @@
+#!/bin/bash
+
+set -eo pipefail
+
+pushd "$(dirname "${BASH_SOURCE[0]}")/.." > /dev/null
+
+BUILD_ARGS=(
+    --product swiftlint
+    --configuration release
+    -Xlinker -S
+)
```

Comment:
Are the other linker options not needed anymore?

Reply:
Also still playing around here. The main issue I'm having at the moment is getting libc statically linked while keeping the other dependencies dynamically linked. `--swift-sdk` is seemingly always producing a fully statically linked binary.